examples/plotting/mplcairo_plotting.py

```diff
@@ -0,0 +1,57 @@
+"""
+==========================================
+Blending Plots using mplcairo
+==========================================
+
+This example will go through how you can blend two plots from SunPy
+using mplcairo. ``matplotlib`` offers solely alpha-based transparency
+for superimposing an image onto another, which can be restrictive
+when trying to create visually appealing composite images. In
+contrast, ``mplcairo`` provides a wide range of blending operators
+for image overlays.
+"""
+
+import matplotlib; matplotlib.use("module://mplcairo.qt")
+import matplotlib.pyplot as plt
+from mplcairo import operator_t
+
+import astropy.units as u
+
+import sunpy.map
+import sunpy.data.sample
+from sunpy.coordinates import Helioprojective
```

Comment:
Rewrite the import section so that it passes pre-commit. It can be ignored (I think) but it helps in knowing what it is. It basically checks if your formatting is similar to the codebase's format and it suggests changes accordingly, the changes it suggested mostly changed the import section.

Try it like this and see if Pre-Comit fails again, you can see the required changes from [here](https://results.pre-commit.ci/run/github/2165383/1678540819.IMxJ4wz-QyCSvANQrAkeqg)

```
from sunpy.coordinates import Helioprojective
import sunpy.map
import sunpy.data.sample
import astropy.units as u
from mplcairo import operator_t
import matplotlib.pyplot as plt
import matplotlib
matplotlib.use("module://mplcairo.qt")
```